### PR TITLE
Copyediting

### PR DESCRIPTION
**Pull request by ahaynes-ador**
:green_book: 

Kathy can thumb and merge at will.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -9,60 +9,59 @@ Thank you for your interest in contributing to LMHT! Enclosed are guidelines for
...
+Before submitting an issue, please visit the LMHT repository's [Issues page] to verify it is not already a known issue. If is is a new issue, click "New Issue" in the upper right corner and title it according to what the issue is. Issues are tracked by title so make it as accurate as possible. Then leave a detailed comment about what the issue is, where it occurs, what button or feature you were trying to use, what did not work correctly, how should it have worked, etc. If possible, submit screen shots or any applicable files to document the issue.  Please do not assign any milestones or labels&mdash;we will do that.
```
Comment (ericanaughton) [inline contributing.md]: a few typos...

"If <strong>is</strong> is a new issue" should be "If <strong>it</strong> is a new issue"

"are tracked by title <strong>so</strong> make it as accurate" should be "are tracked by title <strong>to</strong> make it as accurate"

File: contributing.md [hunk 2]
```diff
@@ -9,60 +9,59 @@ Thank you for your interest in contributing to LMHT! Enclosed are guidelines for
...
 
 ## Request a Feature
 
-Currently, we are using the Issues page to also make feature requests.  Before making a feature request, please review the website and make sure that this feature is in line with the direction the site is going and is a new request.  It is up to you to submit a strong case for getting this feature approved.  If so, then follow the directions above for an issue, and submit your feature request.  Be as detailed as possible about what you want this feature to do.  We will review your case and decide accordingly.  Not all requests will be approved.
+Currently, we are using the [Issues page] to also make feature requests. Before making a feature request, please review the website and make sure that this feature is in line with the site's direction and it is not a repeat request. It is up to you to submit a strong case for approving this feature.
 
+To submit your feature request, follow the directions [in the above seciton](#submit-an-issue) for sumitting an issue. Be as detailed as possible about what you want this feature to do. We will review your case and decide accordingly. Not all requests will be approved.
```
Comment (ericanaughton) [inline contributing.md]: line 31: change "in the above <strong>seciton</strong>" to "in the above <strong>section</strong>"
line 31: change "<strong>sumitting</strong> an issue." to "<strong>submitting</strong> an issue."

File: contributing.md [hunk 3]
```diff
@@ -9,60 +9,59 @@ Thank you for your interest in contributing to LMHT! Enclosed are guidelines for
...
+* Which areas of expertise you wish to contribute (e.g. English to Spanish translation, implementation, documentation)
+* Your level of comfort using GitHub
 
-  1.  Fork the repository and clone it locally.  Attach your local to the original "upstream" respository by additing it as a remote.  Pull the changes from upstream daily so that you are modifying current data when you submit a pull request.  More in-depth instructions [here].
+As a contributor the expectation is that you can use Git and GitHub comfortably enough to fork branches, make pull requests, and practice good commit hygiene.
 
-  2.  Create a branch for your edits.  Depending on what you have loaded in your editor, 'git checkout -b name-your-branch-here, should suffice.
+(These directions are straight from [Contributing to Open Source])
+
+1.  Fork the repository and clone it locally. Attach your local to the original "upstream" respository by additing it as a remote. Pull the changes from upstream daily so that you are modifying current data when you submit a pull request. More in-depth instructions [here].
```
Comment (ericanaughton) [inline contributing.md]: 'additing' should be 'adding'?

**Comment by ericanaughton** [comment]
good job @kathyoleary @JenEEsquivel @LipGlossary !

**Comment by ahaynes-ador** [comment]
Thanks for the keen eye, @ericanaughton.

**Pull request merged**